pkg/users/utils.js

```diff
@@ -12,3 +12,19 @@ export const get_locked = name =>
                     console.warn(`Failed to obtain account lock information for ${name}`, exc);
                 }
             });
+
+async function getLastlogPath() {
+    try {
+        await cockpit.spawn(["test", "-e", "/var/run/utmp"], { err: "ignore" });
+        return "/var/run/utmp";
+    } catch (err1) {
+        try {
+            await cockpit.spawn(["test", "-e", "/var/lib/lastlog/lastlog2.db"], { err: "ignore" });
+            return "/var/lib/lastlog/lastlog2.db";
+        } catch (err2) {
+            return null;
+        }
+    }
+}
+
+export { getLastlogPath, get_locked };
```

Comment:
Is there a reason you changed how functions are exported?  The rest of our code just does `export function...` and I think we should stick with that.

Reply:
I just went with the more Javascript way, i can revert it to the old behaviour. there's really no difference between doing it this way

Reply:
Yes, please revert, just because the rest of Cockpit uses `export function`.